src/h5web/providers/mock/data.ts

```diff
@@ -271,6 +272,15 @@ export const mockMetadata = makeMetadata({
         makeGroupHardLink('log_spectrum'),
       ]
     ),
+    makeGroup('invalid', undefined, [
+      makeGroupHardLink('default_not_string'),
+      makeGroupHardLink('default_empty'),
+      makeGroupHardLink('default_not_found'),
+      makeGroupHardLink('no_signal'),
+      makeGroupHardLink('signal_not_string'),
+      makeGroupHardLink('signal_dataset_not_numeric'),
+      makeGroupHardLink('signal_dataset_not_simple'),
```

Comment:
The only error not covered here is the dimension check in `NXSpectrum`'s `supportsEntity`. I wasn't sure how to create a dataset with zero dimension. A value of `[]` would already indicate one dimension, so it wouldn't make much sense.